spec/bolt/inventory/inventory2_spec.rb

```diff
@@ -387,14 +391,15 @@ def get_target(inventory, name, alia = nil)
 
         it 'should return config for the target' do
           target = get_target(inventory, 'target3')
+          # Should name continue to be duplicated in options? Seems uneccesary
```

Comment:
It seems like the purpose of options is to capture connection configuration for the target, which I don't think includes `name`. So I think we can leave it out.

Reply:
Oops. this comment is from an earlier iteration. Name is not stored in options. My bad. deleted.